runtime/common/BaseRemoteRESTQPU.h

```diff
@@ -290,6 +296,19 @@ class BaseRemoteRESTQPU : public cudaq::QPU {
     passPipelineConfig = std::string("cc-loop-unroll{allow-early-exit=") +
                          allowEarlyExitSetting + "}," + passPipelineConfig;
 
+    if (getEnvBool("DISABLE_QUBIT_MANAGEMENT", false)) {
```

Comment:
@atgeller Is this still valid, or should it be removed?

Reply:
That should probably be removed or at least updated. Can we add an `fno-qubit-mgmt` option to `nvq++` instead perhaps?

Reply:
> That should probably be removed or at least updated. Can we add an `fno-qubit-mgmt` option to `nvq++` instead perhaps?

I don't think that is necessary because `qubit-mgmt` is not being run unless a user explicitly chooses the `opt-test` target with the `dep-analysis` option.

Reply:
Then it should be removed.

Reply:
Removed in 82671ba68